source/Generic/SteamViewer/SteamViewer.psm1

```diff
@@ -7,36 +7,36 @@ function GetGameMenuItems
-    $menuItem10.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentDownloadsDescription")
-    $menuItem10.FunctionName = "Start-ComponentDownloads"
+    $menuItem10.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentActivateProductDescription")
+    $menuItem10.FunctionName = "Start-ComponentActivateProduct"
     $menuItem10.MenuSection = "$extensionName|$subSection"
-    
+
     $menuItem11 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
-    $menuItem11.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentFriendsDescription")
-    $menuItem11.FunctionName = "Start-ComponentFriends"
+    $menuItem11.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentDownloadsDescription")
+    $menuItem11.FunctionName = "Start-ComponentDownloads"
     $menuItem11.MenuSection = "$extensionName|$subSection"
-
+    
     $menuItem12 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
-    $menuItem12.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentNewsDescription")
-    $menuItem12.FunctionName = "Start-ComponentNews"
+    $menuItem12.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentFriendsDescription")
+    $menuItem12.FunctionName = "Start-ComponentFriends"
     $menuItem12.MenuSection = "$extensionName|$subSection"
 
     $menuItem13 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
-    $menuItem13.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentScreenshotsDescription")
-    $menuItem13.FunctionName = "Start-ComponentScreenshots"
+    $menuItem13.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentNewsDescription")
+    $menuItem13.FunctionName = "Start-ComponentNews"
     $menuItem13.MenuSection = "$extensionName|$subSection"
 
     $menuItem14 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
-    $menuItem14.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentSettingsDescription")
-    $menuItem14.FunctionName = "Start-ComponentSettings"
+    $menuItem14.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentScreenshotsDescription")
+    $menuItem14.FunctionName = "Start-ComponentScreenshots"
     $menuItem14.MenuSection = "$extensionName|$subSection"
 
+    $menuItem15 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
+    $menuItem15.Description =  [Playnite.SDK.ResourceProvider]::GetString("LOCSteam_Viewer_MenuItemComponentSettingsDescription")
+    $menuItem15.FunctionName = "Start-ComponentSettings"
+    $menuItem15.MenuSection = "$extensionName|$subSection"
+
     $game = $scriptGameMenuItemActionArgs.Games | Select-Object -last 1
     if ([Playnite.SDK.BuiltinExtensions]::GetExtensionFromId($game.PluginId) -eq "SteamLibrary")
     {
```

Comment:
The new menu item should be added last to prevent modifying all other ones. Also helps to maintain consistency in items placement